### ISSUE TITLE
Glitches (empty frames) in mg printing

### ISSUE THREAD

**Issue by aleksati** [fourMs/AudioVideoAnalysis#21]
This happens even when displaying the regular image, so even with no motion averaging or noise reduction happening. Have a feeling its related to issue #18 . An easy fix might be to scale the x-count printing to be slightly slower than the average framerate of the [grab] objects.. or something.. They are about equal now. Will look into this more and try to find a solution. 

![Annotation 2020-04-13 131321](https://user-images.githubusercontent.com/54307488/79116535-0c629400-7d89-11ea-8e29-2e26ce1f7864.jpg)

**Comment by aleksati** [comment]
Adjusting the camera format and resolution seems to help here. I guess it depends on the machine and battery conditions (etc.). Similar to what we experienced with VideoAnalysis.

In light of this, I added options in the UI to configure the native web-camera format (dimensions). On my Windows machine, I get a number of resolution and formatting options through the jit.grab. It might not be similar on other machines and OS`es, so we have to test this.
